src/peer-store/README.md

```diff
@@ -1,3 +1,156 @@
 # Peerstore
 
-WIP
\ No newline at end of file
+Libp2p's Peerstore is responsible for keeping an updated register with the relevant information of the known peers. It should gather environment changes and be able to take decisions and notice interested parties of relevant changes. The Peerstore comprises four main components: `addressBook`, `keyBook`, `protocolBook` and `metadataBook`. These book components have similar characteristics with the `Javascript Map` implementation.
+
+The PeerStore needs to manage the high level operations on its inner books, have a job runner to trigger other books runners for data trimming or computations. Moreover, the peerStore should be responsible for noticing interested parties of relevant events, through its Event Emitter.
+
+## Peers Environment
```

Comment:
Yeah, still thinking on a good naming. 

The goal in this section is to describe the picture of our network environment. What happens behind the scene to have peers finding out data of other peers (protocols, new multiaddrs, ...) and how they can act with this data via the PeerStore.

The current naming is:

Peers Environment
- Sense
- Act

We can probably move to something like:

Network environment:
- Data Gathering
- Data Consumption

Do you have suggestions?

Reply:
It might be best to just drop the `Peers Environment` header altogether and hoist the subsections to header level 2. We can look at renaming them later. I think what you have listed is better than `sense`and `act` so we can go with that for now.